packages/ui-core/src/components/mouse-positioned-tooltip/mouse-positioned-tooltip.tsx

```diff
@@ -18,58 +21,71 @@ type TooltipContentProps = {
   children: React.ReactNode;
   tooltipVisible: boolean;
   tooltipTheme: TooltipMode;
+  tooltipPinPlacement?: TooltipPinPlacements;
 };
 
 const TooltipContent = ({
   tooltipPosition,
   children,
   tooltipVisible,
   tooltipTheme,
-}: TooltipContentProps) => (
-  <AnimatePresence>
-    {tooltipVisible && (
-      <motion.div
-        {...TOOLTIP_MOTION}
-        initial={{
-          opacity: 0,
-          x: tooltipPosition.x,
-          y: tooltipPosition.y,
-          top: 0,
-          left: 0,
-        }}
-        animate={{
-          x: tooltipPosition.x,
-          y: tooltipPosition.y,
-          opacity: 1,
-        }}
-        style={{
-          position: 'absolute',
-          pointerEvents: 'none',
-          zIndex: UI_LAYERS.tooltip,
-          maxWidth: 225,
-        }}
-      >
-        <Tooltip hasArrow={false} mode={tooltipTheme}>
-          {children}
-        </Tooltip>
-      </motion.div>
-    )}
-  </AnimatePresence>
-);
+  tooltipPinPlacement,
+}: TooltipContentProps) => {
+  const tooltipTranslation = getTooltipTranslation(tooltipPinPlacement);
+
+  return (
+    <AnimatePresence>
+      {tooltipVisible && (
+        <motion.div
+          {...TOOLTIP_MOTION}
+          initial={{
+            opacity: 0,
+            x: tooltipPosition.x,
+            y: tooltipPosition.y,
+            top: 0,
+            left: 0,
+          }}
+          animate={{
+            x: tooltipPosition.x,
+            y: tooltipPosition.y,
+            opacity: 1,
+          }}
+          style={{
+            position: 'absolute',
+            pointerEvents: 'none',
+            zIndex: UI_LAYERS.tooltip,
+            maxWidth: 225,
+          }}
+        >
+          <TooltipWrapper
+            translateX={tooltipTranslation.x}
+            translateY={tooltipTranslation.y}
+          >
+            <Tooltip hasArrow={false} mode={tooltipTheme}>
+              {children}
+            </Tooltip>
+          </TooltipWrapper>
+        </motion.div>
+      )}
+    </AnimatePresence>
+  );
+};
 
 type Props = {
   children: React.ReactNode;
   isActive: boolean;
   tooltipPortal?: string;
   tooltipTheme?: TooltipMode;
   renderContent: () => React.ReactNode;
+  tooltipPinPlacement?: TooltipPinPlacements;
 };
 
 const MousePositionedTooltip = ({
   children,
   isActive,
   tooltipPortal,
   tooltipTheme = 'light',
+  tooltipPinPlacement,
```

Comment:
I suggest to use as default `bottom-right` and modify switch case to reflect all possible values. What do You think ?

Reply:
Good idea, i have changed it.